sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -169,6 +172,30 @@ def test_do_with_multiple_outputs_maintains_unique_name(self):
     pipeline.run()
 
   @attr('ValidatesRunner')
+  def test_read_from_text_metrics(self):
+    from apache_beam.examples.snippets.snippets import CountingSource
```

Comment:
I think examples should be completely isolated and nothing inside actual beam code should use anything from the examples package. We should remove this.